SQL/New_patches/2018-01-17-normalisation_visit.sql

```diff
@@ -1,25 +1,27 @@
+ALTER TABLE project_rel DROP PRIMARY KEY, ADD COLUMN `ProjectSubprojectTupleID` int(10) unsigned NOT NULL PRIMARY KEY AUTO_INCREMENT FIRST;
```

Comment:
@ridz1208 Have you tested this line?
Had to split in 2 alter table lines in the past (1-drop PK, 2- add column with PK)

Reply:
I tested it on mysql 5.7

I actually could not split it because mysql refuses to drop a primary key unless you specify another one in the same statement